### PR TITLE
SEO-504: StoriesLinkTag - update conditions to render the tag

### PR DESCRIPTION
**Pull request by saipetch**

### CODE REVIEW

File: extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php [hunk 1]
```diff
@@ -140,7 +140,8 @@ public function render() {
 		$urls = array_slice( array_filter( $urls ), 0, self::MAX_URLS );
 		foreach ( $urls as $url ) {
 			$urlParts = parse_url( trim( $url ) );
-			if ( !empty( $urlParts['host'] ) && strtolower( $urlParts['host'] ) == StoriesLinkTagHelper::VALID_HOST
+			if ( !empty( $urlParts['scheme'] ) && strtolower( $urlParts['scheme'] ) == 'http'
```
Comment (WikiaTech) [inline extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php:143]: MAJOR Reduce the number of conditional operators (5) used in the expression (maximum allowed 3). [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [StoriesLinkTagController.class.php#L66](https://github.com/Wikia/app/blob/628b3a8e443feb6119ab7fa90a0fb2553aa3ce0d/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L66): Immediately return this expression instead of assigning it to the temporary variable "$markerId". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)
1. MINOR [StoriesLinkTagController.class.php#L110](https://github.com/Wikia/app/blob/628b3a8e443feb6119ab7fa90a0fb2553aa3ce0d/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L110): Immediately return this expression instead of assigning it to the temporary variable "$html". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* MAJOR 1 major
* MINOR 2 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [StoriesLinkTagController.class.php#L66](https://github.com/Wikia/app/blob/8a38b266e0dc8c06b2921324d22cd55e762f60ae/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L66): Immediately return this expression instead of assigning it to the temporary variable "$markerId". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)
1. MINOR [StoriesLinkTagController.class.php#L110](https://github.com/Wikia/app/blob/8a38b266e0dc8c06b2921324d22cd55e762f60ae/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L110): Immediately return this expression instead of assigning it to the temporary variable "$html". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)

**Comment by garthwebb** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
- MAJOR 1 major
- MINOR 2 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MINOR [StoriesLinkTagController.class.php#L66](https://github.com/Wikia/app/blob/9764aec6291ca470e5a20a658abeb80881188bd7/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L66): Immediately return this expression instead of assigning it to the temporary variable "$markerId". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)
2. MINOR [StoriesLinkTagController.class.php#L110](https://github.com/Wikia/app/blob/9764aec6291ca470e5a20a658abeb80881188bd7/extensions/wikia/StoriesLinkTag/StoriesLinkTagController.class.php#L110): Immediately return this expression instead of assigning it to the temporary variable "$html". [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)

**Comment by gabrys** [comment]
LGTM

**Pull request merged**